src/config/config_def.c

```diff
@@ -62,7 +62,9 @@ static const WT_CONFIG_CHECK confchk_WT_CONNECTION_reconfigure_compatibility_sub
 static const WT_CONFIG_CHECK confchk_wiredtiger_open_debug_mode_subconfigs[] = {
   {"checkpoint_retention", "int", NULL, "min=0,max=1024", NULL, 0},
   {"corruption_abort", "boolean", NULL, NULL, NULL, 0},
-  {"cursor_copy", "boolean", NULL, NULL, NULL, 0}, {"eviction", "boolean", NULL, NULL, NULL, 0},
+  {"cursor_copy", "boolean", NULL, NULL, NULL, 0},
+  {"cursor_reposition", "boolean", NULL, NULL, NULL, 0},
+  {"eviction", "boolean", NULL, NULL, NULL, 0},
```

Comment:
What's this change for eviction?

Reply:
This is not a new change. It existed before as well. Just on one line. [here](https://github.com/wiredtiger/wiredtiger/pull/7977/files#diff-ff528f30fd5f654ae4d06b4cf237b1f3e4e57be76a432dcbb00e91c13e646d52L65)